lib/inputstreamhelper/__init__.py

```diff
@@ -744,14 +776,16 @@ def _widevine_eula(self):
 
         return Dialog().yesno(localize(30026), eula, yeslabel=localize(30027), nolabel=localize(30028))  # Widevine CDM EULA
 
-    def _extract_widevine_from_img(self):
-        ''' Extract the Widevine CDM binary from the mounted Chrome OS image '''
-        for root, dirs, files in os.walk(str(self._mnt_path())):  # pylint: disable=unused-variable
+    def _extract_widevine_from_img(self, backup_path):
+        ''' Extract the Widevine CDM binary from the mounted Chrome OS image to backup dir '''
+        for root, _, files in os.walk(str(self._mnt_path())):
             if str('libwidevinecdm.so') not in files:
                 continue
             cdm_path = os.path.join(root, 'libwidevinecdm.so')
             log('Found libwidevinecdm.so in {path}', path=cdm_path)
-            shutil.copyfile(cdm_path, os.path.join(self._ia_cdm_path(), 'libwidevinecdm.so'))
+            if not xbmcvfs.exists(backup_path):
+                xbmcvfs.mkdir(backup_path)
+            shutil.copyfile(cdm_path, os.path.join(backup_path, 'libwidevinecdm.so'))
```

Comment:
Does this work on Windows and Mac?

Reply:
`_extract_widevine_from_img()` will only ever be called on ARM devices with Linux. If you compare the deleted line above, this has obviously worked before.

Reply:
Okay, I see.